commit 94b3978aa1d01f09b29dbc2d61e1a2bddec313df
Author: Lukas Herbolt <lherbolt@redhat.com>
Date:   Wed Oct 25 15:47:18 2017 -0500

    Fix BZ1506230 by fixing netdev.receive probe to work on kernels after v3.14.

diff --git a/tapset/linux/networking.stp b/tapset/linux/networking.stp
index 76174c7..17cba9c 100644
--- a/tapset/linux/networking.stp
+++ b/tapset/linux/networking.stp
@@ -59,7 +59,8 @@ function get_netdev_name:string (addr:long) {
 ///</para>
 // Main device receive routine, be called when packet arrives on network device
 probe netdev.receive
-	=  kernel.function("netif_receive_skb")
+	= kernel.function("netif_receive_skb_internal") !,
+	  kernel.function("netif_receive_skb")
 {
 	dev_name = kernel_string($skb->dev->name)
 	length = $skb->len
